src/actions/activeZone.js

```diff
@@ -1,34 +1,34 @@
-import * as ActionTypes from '../constants/ActionTypes';
-import { asyncDNSRecordFetchList } from './zoneDnsRecords';
-import { asyncZoneFetchAnalytics } from './zoneAnalytics';
-import { asyncZoneRailgunFetchAll } from './zoneRailgun';
-import { asyncZoneFetchSettings } from './zoneSettings';
-import { asyncPluginFetchSettings } from './pluginSettings';
```

Comment:
If cf-www-next ends up with `singleQuote: true` I'll go back and update it again.  Just wanted to get this PR out before the UI meeting so we had an example to talk about.